Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.yml

```diff
@@ -753,7 +792,152 @@ script:
       description: Anomaly Name
       type: string
     description: Retrieve Detailed Information for a Given User
-  dockerimage: demisto/python3:3.9.1.14969
+  - name: gra-case-action
+    arguments:
+    - name: action
+      required: true
+      default: true
+      auto: PREDEFINED
+      predefined:
+      - closeCase
+      - modelReviewCase
+      - riskManageCase
+      description: Action
+      defaultValue: closeCase
+    - name: caseId
+      required: true
+      description: Case Id
+    - name: subOption
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - True Incident
+      - Correct Detection
+      - Tuning Required
+      - Others
+      description: Sub Option
+      defaultValue: True Incident
+    - name: caseComment
+      required: true
+      description: Case Comment
+    - name: riskAcceptDate
+      description: Risk Accept Date in ‘yyyy-MM-dd’ format. (applicable only in case
+        of closing a case as Risk Managed)
+    outputs:
+    - contextPath: Gra.Case.Action.Message
+      description: Message
+      type: string
+    description: Closing a case and updating the anomaly status as Closed / Risk Managed
+      / Model Reviewed.
+  - name: gra-case-action-anomaly
+    arguments:
+    - name: action
+      required: true
+      default: true
+      auto: PREDEFINED
+      predefined:
+      - closeCaseAnomaly
+      - modelReviewCaseAnomaly
+      - riskAcceptCaseAnomaly
+      description: Action
+      defaultValue: closeCaseAnomaly
+    - name: caseId
+      required: true
+      description: Case ID
+    - name: anomalyNames
+      required: true
+      description: Anomaly Names
+    - name: subOption
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - True Incident
+      - Correct Detection
+      - Tuning Required
+      - Others
+      description: Sub Option
+      defaultValue: True Incident
+    - name: caseComment
+      required: true
+      description: Case Comment
+    - name: riskAcceptDate
+      description: Risk Accept Date in ‘yyyy-MM-dd’ format. (applicable only in case
+        of closing anomalies as Risk Accepted)
+    outputs:
+    - contextPath: Gra.Case.Action.Anomaly.Message
+      description: Message
+      type: string
+    - contextPath: Gra.Case.Action.Anomaly.Anomaly Name
```

Comment:
Context Keys should not contain spaces. Please fix according to the returned object.

*same for all occurances*